devtools/mog/config.go

```diff
@@ -176,7 +201,7 @@ func parseFieldAnnotation(field *ast.Field) (fieldConfig, error) {
 		case "target":
 			c.TargetName = value
 		case "pointer":
-			// TODO:
+			// TODO(rb): remove as unnecessary?
```

Comment:
The types of options that I expected here were things like:

1. how to deal with zero values when converting to a pointer. Should they be converted to `nil`, or a pointer to the zero value
2. if both the source and target pointers are of the same type, should they reference the same value, or be a copy. But I realize now that is more general than pointers. It should apply to maps and slices as well.